### ISSUE TITLE
Add support for listing "lost" episodes

### ISSUE THREAD

**Issue by Starmann** [lolamtisch/MALSync#242]
After episode 14, the original Mobile Suit Gundam series has an off-by-one error due to the infamously terrible episode 15 essentially being deleted. Animepahe.com counts Episodes using the English version's counting scheme, which subtracts 1 from every episode after 14. This creates some really confusing off-by-one errors that need to manually be corrected for

**Comment by lolamtisch** [comment]
There is a offset functionality for that:
https://github.com/lolamtisch/MALSync/wiki/Troubleshooting#the-series-uses-an-episode-offset
You will need to set it after watching the 14th episode.

**Comment by Starmann** [comment]
Yes I know about offsets. I was wondering if you could make a feature that increases the offset when reached. Until then this is a decent work around.

**Comment by lolamtisch** [comment]
It is planned to add a advance correction mode. Where you can add multiple mal entries with differnet offsets to one streaming entry. So you could define them on episode basis.
But as it is a big restructering of data, so it wont added very soon. 
Probably when it gets added it will still be a manual prozess as restructering the mapping database is even more work.
